tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "files": ["./types/react-table-config.d.ts"],
```

Comment:
I was having an issue with `dashboard` not compiling because it didn't have this. Seems it's required in every package which uses the table. So, made this folder? 🤷

Reply:
yeah, it is required - this is a much nicer solution thanks.